### PR TITLE
Append position to end of error message

### LINKED ISSUE #9: Append line / column of error after file
It came up in [this issue](https://github.com/lerna/lerna/issues/854#issuecomment-318181789) that terminals like iTerm2 for macOS and editors like Sublime Text recognize references to lines and columns appended to a file like this:

    /Users/andrey/code/foo/package.json:7:31

Would you accept a pull request editing the error format and documentation to conform to this style? It would allow folks to get to the exact point of the error with a single click & hence solve it most efficiently.

### PR DESCRIPTION
**Pull request by anfedorov**
Fixes #9 
Closes #10

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -2,8 +2,20 @@
...
+	return message.replace(posRe, ' in') + ':' + numbers[1];
+}
+
 const JSONError = errorEx('JSONError', {
-	fileName: errorEx.append('in %s')
+	fileName: errorEx.append('in %s'),
+	appendPosition: {
+		message: (shouldAppend, original) => {
+			const originalMessage = original[0];
+			return shouldAppend ? appendPosition(originalMessage) : originalMessage;
```
Comment (Qix-) [inline index.js]: Should honor multi-line error messages just to be safe.

```javascript
if (shouldAppend) {
    original[0] = appendPosition(original[0]);
}

return original;
```

Comment (anfedorov) [inline index.js]: Good call. Should I squash the two commits?

**Comment by sindresorhus** [comment]
Travis is failing

File: index.js (hunk 1 above)
Comment (sindresorhus) [inline index.js]: Nah, I'll squash on merge.

**Comment by anfedorov** [comment]
fixed — had editor set on spaces :-)

**Pull request merged**